src/renderer/services/ipc/index.js

```diff
@@ -0,0 +1,49 @@
+import { uniqueId } from 'lodash';
+
+const { ipcRenderer } = window && window.electron;
```

Comment:
Is there a reason why don't you put the code in `common` ?

Reply:
which code ?